pandas/io/excel/_xlsxwriter.py

```diff
@@ -8,7 +10,7 @@ class _XlsxStyler:
     # Map from openpyxl-oriented styles to flatter xlsxwriter representation
     # Ordering necessary for both determinism and because some are keyed by
     # prefixes of others.
-    STYLE_MAPPING = {
+    STYLE_MAPPING: Dict[str, list] = {
```

Comment:
list -> List and add type parameters.

I think `Dict[str, List[Tuple[Tuple[str, ...], str]]]`